Resources/ServerInfo/_CP14/Guidebook_EN/JobsTabs/Innkeepers.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+  # Innkeeping
+  The innkeeper is a role that primarily stays within the town. The responsibilities of an innkeeper include gardening, bartending, and cooking.
+
+  Within the inn, an innkeeper will have access to three main areas:
+
+  ## The Garden
+  The garden, or farm, is an important asset for any chef. A few seedbeds can be found here, along with a well and some basic supplies.
+  To begin farming, locate the supply table in the garden. Usually, a crate will contain a variety of seeds, while a table will hold farming tools like a bucket and hoe. Grab a seed variety and use it on the ground to plant it. Wait a while, then use any sharp tool (such as a sickle or dagger) to harvest the fully grown crop.
+
+  To keep your crops alive for an extended period, be sure to water them frequently. Keeping plants healthy is easier with the spell **“Plant Growth”** from the Vivification tree, which restores a plant’s water, nutrients, and health.
+
+  If all your seedbeds are full and demand is still high, you can use a hoe to till additional soil. Consider making a seed bag if you plan to expand your farm.
+
+  ## The Cellar
+  The cellar is extremely popular among patrons. Several barrels can be found here that dispense alcohol. Grab a mug, ensure you are drawing from a keg, and profit.
+
+  A typical cellar also contains a few tables and torches, as well as a barrel of water. Tables usually hold spare mugs. Be careful not to serve too much alcohol at once... unless your esteemed customer is a dwarf.
+
+  ## The Kitchen
+
+  Most of an innkeeper's time is spent in the kitchen. First, gather materials: chop or find some wood for planks, start growing crops, locate spare food items, and grab your tools. A good chef will recognize the value of a dagger — typically provided to an innkeeper — which can be used to harvest plants, chop food into slices, and fend off unruly patrons.
+
+  Let’s begin with cooking something simple: a green salad. Start by growing cabbage, pumpkin, and cucumber. Once the crops are ready, bring them into the kitchen. Go to your cooking table and find the recipe for green salad. You may notice the recipe requires sliced ingredients — simply use your knife to prepare them. Next, find a plate (usually some are stored near the ovens). Once all ingredients are near the table, you can begin to assemble the salad.
+
+  Green salad is easy to make because it doesn't require heating. However, almost all other dishes will need to be cooked using the furnace. To use it, place 2–3 wood planks inside to keep the fire burning long enough. Ignite the fire with either the **“Create Flame”** spell or a flint and steel. Then, place the uncooked item on top of the furnace — *after* lighting it — so it begins to cook. Remove the food promptly, or it will burn into inedible ash (which you might try to market, but it won’t end well). The **“Freeze”** spell can stop a furnace from wasting fuel between cooking sessions.
```

Comment:
The "Heat" spell will allow you to instantly cook food without using up scarce wood.

It is better to use wood in the Workbench to craft MUGS and PLATES.